### PR TITLE
[ONNX] gru rnn v3

### PR DESCRIPTION
**Pull request by arogowie-intel**
- [ ] unit tests for GRU
- [ ] unit tests for RNN

### CODE REVIEW

File: src/ngraph/op/fused/lstm_sequence.hpp [hunk 1]
```diff
@@ -66,7 +60,7 @@ namespace ngraph
                                       const Output<Node>& B,
                                       const Output<Node>& P,
                                       const std::int64_t hidden_size,
-                                      const direction lstm_direction,
+                                      const RecurrentSequenceDirection lstm_direction,
```
Comment (silee2) [inline src/ngraph/op/fused/lstm_sequence.hpp:63]: Why do you need an API breaking change?

Comment (diyessi) [reply src/ngraph/op/fused/lstm_sequence.hpp:63]: Add a `using direction=RecurrentSequenceDirection` to the class so it won't be API-breaking.

Comment (arogowie-intel) [reply src/ngraph/op/fused/lstm_sequence.hpp]: I need some enum to use for ONNX's RNN and GRU direction mode. Added using statement.

File: src/ngraph/frontend/onnx_import/utils/recurrent.cpp [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+                        "RecurrentSequence: unhandled direction mode during decomposition.");
+                }
+                return results;
+            }
+
+            NodeVector
+                RecurrentSequence::recurrent_sequence_pass(const RecurrentCellFunction& kernel,
+                                                           bool is_reverse)
+            {
+                NodeVector h_list;
```
Comment (rkimballn1) [inline src/ngraph/frontend/onnx_import/utils/recurrent.cpp:163]:
```suggestion
                OutputVector h_list;
```

**Comment by arogowie-intel** [comment]
Moved to OpenVINO repository.

**Pull request closed without merging**